src/state/store/initialState.js

```diff
@@ -0,0 +1,3 @@
+const initialState = {};
+
+export default initialState;
```

Comment:
Well, if the initial state is going to be empty, then you don't have to use it at all.

Reply:
File removed.